hack/make-rules/tools.mk

```diff
@@ -115,20 +120,16 @@ $(TOOLBIN)/crdoc:
 	GOBIN=$(ABSTOOLBIN) go install fybrik.io/crdoc@v0.6.1
 	$(call post-install-check)
 
-INSTALL_TOOLS += $(TOOLBIN)/json-schema-generator
-$(TOOLBIN)/json-schema-generator:
+INSTALL_TOOLS += json-schema-generator
+json-schema-generator:
 	cd $(TOOLS_DIR); ./install_json-schema-generator.sh
 	$(call post-install-check)
 
-INSTALL_TOOLS += $(TOOLBIN)/yq
-$(TOOLBIN)/yq:
```

Comment:
I haven't removed `$(TOOLBIN)/yq`, Just moved it to the top of the file because other installation depends on it.
As you can see on [my local workflow run](https://github.com/Raviv-S/fybrik/runs/7409299728?check_suite_focus=true), it doesn't fail on that stage.
I'm not sure why it failed on the first time but I think a re-run will fix it.